stack/core/src/main/java/org/apache/usergrid/persistence/cassandra/CassandraService.java

```diff
@@ -230,7 +239,7 @@ public void setConsistencyLevelPolicy( ConsistencyLevelPolicy consistencyLevelPo
     /** @return keyspace for application UUID */
     public static String keyspaceForApplication( UUID applicationId ) {
         if ( USE_VIRTUAL_KEYSPACES ) {
```

Comment:
Just change this code, we always use virtual keyspaces, so the other scheme isn't valid any longer